### PR TITLE
Add docs and release notes for 6.16 about CORS changes

### PR DESCRIPTION
**Pull request by mkuznyetsov**
### What does this PR do?
Adding docs and release notes for 6.16 describing changes to CORS (https://github.com/eclipse/che/pull/12144)

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/12127

### CODE REVIEW

**Comment by mkuznyetsov** [comment]
@rkratky @dneary can you review this please?

File: release-notes/release-6.16.0.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Upgrading
+
+Instructions on how to upgrade.
+
+
+## Release details
+
+### Upgrading Tomcat & CORS configuration (https://github.com/eclipse/che/pull/12144)
+
+In this release, Tomcat  will be upgraded to newest version 8.5.35, alongisde our CORS configuration to be compatible with it.
```
Comment (dneary) [inline release-notes/release-6.16.0.md:16]: I could use a description of what is CORS, and how this makes the Che user experience better. This is unclear to me.

Thanks!

File: release-notes/release-6.16.0.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+Here is the full list of environment variables that will be available in Che 6.16 for overriding of default configuration for CORS on WS Master and WS Agent: 
+
+* `CHE_CORS_ENABLED` = Enabling CORS Filter on WS Master. On by default
+* `CHE_CORS_ALLOWED_ORIGINS` = List of allowed origins in requests to WS Master. Default is "*".
+* `CHE_CORS_ALLOW_CREDENTIALS` = Allowing requests with credentials to WS Master. Default is "false".
+
+* `CHE_WSAGENT_CORS_ENABLED` = Enabling CORS Filter on WS Agent. 
+* `CHE_WSAGENT_CORS_ALLOWED__ORIGINS` = List of allowed origins in requests to WS Agent. If not set, or set to "null", value will be evaluated from CHE_API variable at runtime.
+* `CHE_WSAGENT_CORS_ALLOW__CREDENTIALS` = Allowing requests with credentials to WS Agent. Default is "true".
+
```
Comment (dneary) [inline release-notes/release-6.16.0.md:42]: Can you add links to the PRs which cover these changes, please?

Comment (mkuznyetsov) [reply release-notes/release-6.16.0.md:42]: added

File: release-notes/release-6.16.0.md (hunk 1 above)
Comment (mkuznyetsov) [reply release-notes/release-6.16.0.md:16]: added sentence about that

**Review by dneary — APPROVED** [review]
Looks good to me. Any other changes can be made in the RN file.

**Pull request merged**